blog/notatki-graficzne/index.md

```diff
@@ -0,0 +1,149 @@
+
+W marcu 2023 wziąłem udział w szkoleniu prowadzonym przez
+[Gabrielę Borowczyk](https://www.linkedin.com/in/gabriela-borowczyk-%E2%9C%8F-55558126/).
+Otrzymałem solidną porcję wiedzy o technikach tworzenia różnego rodzaju napisów,
+elementów dekoracyjnych, liter i rysunków. Są to umiejętności pomocne zwłaszcza
+dla scrum masterów podsumowujących wizualnie spotkania, warsztaty i spontaniczne
+burze mózgów. Szkolenie, w którym wziąłem udział z innymi pracownikami Hitachi
+Energy, było bardzo sympatyczne i intensywne. Mieliśmy okazję podkręcić
+kreatywność na wyższe obroty. Ubawiłem się przednio, szczególnie, że ostatni raz
+z takim zapałem rysowałem w liceum, a później było coraz mniej do tego okazji.
+Sam temat był dla mnie ważny o tyle, że moją odwieczną bolączką był charakter
+pisma zbliżony do szyfru lub recepty wypisanej przez lekarza. Było to krzepiące,
+że mimo mojego awangardowego stylu pisania, również i ja byłem w stanie się
+nauczyć jak pisać na plakatach dostatecznie wyraźnie. Dowiedziałem się też o
+graficznych facylitatorach prowadzących warsztaty z użyciem własnoręcznie
+narysowanych materiałów, a także o ilustratorach spotkań i wydarzeń, tworzących
+wizualne podsumowania, które są świetnymi materiałami edukacyjnymi.
+
+## I co dalej zrobić z tą wiedzą?
+
+![](images/upload_0087.png) Warsztaty ukończyłem, dyplom otrzymałem. Przez
+dłuższy czas nie tykałem flamastra, aż pewnego pięknego dnia stwierdziłem, że
+wypróbuję te umiejętności w codziennej praktyce speca od dokumentacji i autora
+radosnej twórczości po godzinach. Poszerzyłem mój zestaw pisaków, zakupiłem
+[biuwar (moje nowe ulubione słowo ;)](https://sjp.pwn.pl/slowniki/biuwar.html) i
+zacząłem eksperymenty z pisanymi i rysowanymi notatkami. Dla mniej
+zorientowanych, biuwar to duży arkusz papieru z kalendarzem, miejscem na
+odręczne rysunki i listę zadań.
+
+## Jak korzystam z wizualnych notatek
+
+![](images/upload_0088.png) Zarówno w pracy jak i po godzinach tworzę listy
+zadań, mapy tematów i sieć połączeń miedzy nimi. Rysunki i zapiski pomagają mi
+spamiętać rzeczy, które aktualnie biorę na tapet. Zapisuję sobie główne tagi,
+którymi kategoryzuję notatki w formacie markdown, w narzędziu Obsidian, któremu poświęcę
+przyszły artykuł :) Rysunki "pączkują" i z biegiem czasu tworzy się mozaika.
+Jeśli tematy znajdują się blisko siebie, znaczy to, że są w jakiś sposób ze sobą
+związane. Za pomocą kolorów i innych metod mogę podkreślać powiązane tematy. Z
+lubością wykreślam zamknięte sprawy. Kiedy na arkuszu robi się zbyt tłoczno albo
+większość kwestii jest rozwiązanych, wydzieram kartkę i biorę się do zapisywania
+kolejnej. Stare zapiski zachowuję, bo stanowią wizualny zapis stanu rzeczy z
+danego momentu projektu. Czasem do pewnych szczegółów warto wrócić.
+
+## W czym pomagają mi wizualne notatki
+
+Notatki wizualne wspierają mnie w zapamiętywaniu nazw projektów, komponentów,
+dat i powiązań. Tak jak notatka na lodówce, wzrok często spoczywa na arkuszu i
```

Comment:
To zdanie nie jest zrozumiałe i chyba niepoprawne. Cz chodziło o coś takiego?
```suggestion
dat i powiązań. Arkusz z zapiskami jest jak notatka na lodówce. Wystarczy na nią spojrzeć, żeby odświeżyć sobie kluczowe informacje.
```

Reply:
nowa wersja jest ok. chodzilo mi o to, ze rzecz jest na widoku i mimowolnie powtarzasz sobie informacje, nawet jak swiadomie nie potrzebujesz sprawdzic tej informacji. Ale nie jest to niezbedny niuans.

Reply:
Aaaa, no to teraz wszystko jasne. Myślę, że ten niuans jest istotny, dlatego zmieniłem swoją sugestię. Zobacz czy teraz jest lepiej :)